### ISSUE TITLE
Accuarcy about NIN on CIFAR10 dataset

### ISSUE THREAD

**Issue by fuyuanlyu** [jiecaoyu/XNOR-Net-PyTorch#59]
Dear Author:

Hi! I write to ask about the accuracy rate on your implemented NIN on cifar10 dataset.

I'm running your code and adapt some hyper-parameter like learning rate, batch size and learning rate decay(when to times 0.1). However, the best validation accuracy is 81%. 

If you don't mind, may I ask the hyper-parameter your set to get the 86% accuracy? Or is it just like the file you uploaded? It would be great help to me.

One more thing, if you don't mind, I'd like to cite this project. Are there any paper or homepage regarding it? Or just this page?

FYI, great project!

Yours,
Fuyuan

**Comment by jiecaoyu** [comment]
Hi @silentspring2 , can you try using a initial learning rate of 0.01? It seems work for me. I am running the experiment and will upload the training log later.

For citation, please just cite [this pdf](https://github.com/jiecaoyu/XNOR-Net-PyTorch/blob/master/notes/notes.pdf). Thanks!

**Comment by fuyuanlyu** [comment]
Thanks for your quick reply, I'll try to set lr to 0.01 and inform you further after the experiment.
I'll add the pdf to my citation.

Thanks!

Yours,
Fuyuan

**Comment by jiecaoyu** [comment]
Hi @silentspring2 , you can download the training log [here](https://drive.google.com/file/d/18dkMyn3OwXh7jp4k7zxf_0kurhgc9gO3/view?usp=sharing).

**Comment by fuyuanlyu** [comment]
Thank you very much for your help!
